themes/cucumber-hugo/layouts/shortcodes/card.html

```diff
@@ -1,4 +1,4 @@
-<a href="http://google.com">
+<a href="{{ .Get "link" }}">
   <div class="card" style="height: 100%">
```

Comment:
Going to a specific link per book instead of Google home page is a great idea. Just not sure about what the particular links should be